src/Console/Commands/AuthMigration.php

```diff
@@ -29,6 +30,12 @@ public function handle()
 
         copy($from, $to);
 
+        $contents = str_replace('USERS_TABLE', config('statamic.users.tables.users', 'users'), File::get($to));
+        $contents = str_replace('ROLE_USER_TABLE', config('statamic.users.tables.role_user', 'role_user'), $contents);
+        $contents = str_replace('GROUP_USER_TABLE', config('statamic.users.tables.group_user', 'group_user'), $contents);
+
+        File::put($to, $contents);
+
```

Comment:
Instead of `copy`ing the file and then re-writing it, just `File::get($from)` and then `File::put()` it in `$to`.